components/prism-diff.js

```diff
@@ -39,11 +39,28 @@
 			alias.push("bold");
 		}
 
-		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
-			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
-		};
+		var pattern = RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm');
+
+		if (['-', '<', '+', '>'].indexOf(prefix) !== -1) {
+			var inside = {
+				line: {
+					pattern: /^(.).+(?:\r\n?|\n)?/,
```

Comment:
Sorry, I made a mistake here. This pattern has problems with empty lines (no chars after the prefix).
Here's the fix:

```suggestion
					pattern: /(.)(?=[\s\S]).*(?:\r\n?|\n)?/,
```